f5_openstack_agent/lbaasv2/drivers/bigip/network_helper.py

```diff
@@ -514,25 +559,38 @@ def add_fdb_entry(
         else:
             records.append(fdb_entry)
 
-        tunnel = bigip.net.fdbs.tunnels.tunnel
-        if tunnel.exists(name=tunnel_name, partition=partition):
-            tunnel.load(name=tunnel_name, partition=partition)
-            tunnel.update(records=records)
-            if const.FDB_POPULATE_STATIC_ARP:
-                if arp_ip_address:
-                    try:
-                        arp = bigip.net.arps.arp
-                        arp.create(ip_address=arp_ip_address,
-                                   mac_address=mac_address,
-                                   partition=partition)
-                    except Exception as e:
-                        LOG.error('add_fdb_entry',
-                                  'could not create static arp: %s'
-                                  % e.message)
-                        return False
-            return True
+        try:
+            tunnel = bigip.net.fdbs.tunnels.tunnel
+            if tunnel.exists(name=tunnel_name, partition=partition):
+                tunnel.load(name=tunnel_name, partition=partition)
+                tunnel.update(records=records)
+                if const.FDB_POPULATE_STATIC_ARP:
+                    if arp_ip_address:
```

Comment:
This is always None?  (i.e. arp_ip_address)  This i guess is the ip address of the member.